src/index.html

```diff
@@ -184,5 +194,64 @@ <h2 class="modal__title">New Post</h2>
       </div>
     </div>
     <!-- <script type="module" src="./scripts/index.js"></script> -->
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button
+          type="button"
+          class="modal__close"
+          aria-label="Open modal"
+        ></button>
+        <h3 class="modal__title">Edit profile avatar</h3>
+        <form
+          name="edit-avatar-form"
+          class="modal__form"
+          id="edit-avatar-form"
+          novalidate
+        >
+          <label for="profile-avatar-input" class="modal__label"
+            >Profile Picture
+            <input
+              id="profile-avatar-input"
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/Z9ni6f.jpg